frontend/packages/dev-console/src/components/topology/__tests__/topology-utils.spec.ts

```diff
@@ -266,6 +271,47 @@ describe('TopologyUtils ', () => {
     ]);
     expect((topologyTransformedData[keys[2]].data as WorkloadData).builderImage).toBe(csvIcon);
   });
+
+  it('should not render event sources if corresponding filter returns false', () => {
+    const eventFilter: TopologyFilters = {
+      display: {
+        podCount: true,
+        eventSources: false,
+        knativeServices: true,
+        appGrouping: true,
+        operatorGrouping: true,
+      },
+      searchQuery: '',
+    };
+    const { topologyTransformedData } = getTranformedTopologyData(
+      MockKnativeResources,
+      [],
+      '',
+      eventFilter,
+    );
+    expect(topologyTransformedData['1317f615-9636-11e9-b134-06a61d886b689']).toBe(undefined);
+  });
+  it('should render event sources if corresponding filter returns true', () => {
+    const eventFilter: TopologyFilters = {
+      display: {
+        podCount: true,
+        eventSources: true,
+        knativeServices: true,
+        appGrouping: true,
+        operatorGrouping: true,
+      },
+      searchQuery: '',
+    };
```

Comment:
declare it once either in beforeEach or out of `describe` block. and you should be importing this from `redux/const.ts`
and do something like this
`const topologyFilters = {...DEFAULT_TOPOLOGY_FILTERS};`
and change the topology filter in each test case as you need it.

Reply:
I've added the changes. PTAL